runtime/cudaq/platform/default/rest/helpers/equal1/Equal1ServerHelper.cpp

```diff
@@ -111,6 +114,11 @@ void Equal1ServerHelper::initialize(BackendConfig config) {
 
   devicePassPipeline = queryPassPipeline();
 
+  if (config.count("emulate") && config["emulate"] == "false")
+    devicePassPipeline = queryPassPipeline();
+  else
+    devicePassPipeline = "canonicalize";
```

Comment:
can we put "const-prop-complex, canonicalize, cse" as the pipeline? It can make IR more readable if we will get to debug it